README.md

```diff
@@ -18,7 +18,7 @@ _Collaborate and work together on GitHub._
   Do not use quotes on the <details> tag attributes.
 -->
 
-<details id=0 open>
+<details id=0>
```

Comment:
Is this the edit that was added to the file?

Reply:
Yes by the automation once this Step is "Completed" Notice the <open> field/value in the "-"/deleted entry and in the added "+" edit <open> is not indicated..